conf/docker.properties.template

```diff
@@ -0,0 +1,3 @@
+spark.executor.docker.image: amplab/spark-1.1.0
```

Comment:
Should this be hardcoded ? And if so, shouldn't this default be documented ? (`docs/running-on-mesos` could mention it)